evap/static/maintenance/maintenance.html

```diff
@@ -0,0 +1,36 @@
+<html>
+	<head>
+		<meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+        <title>EvaP</title>
```

Comment:
mixed indentation, these lines use spaces, above and below uses tabs. Indentation is also weird in 14, 25 and 26.